### ISSUE TITLE
Users should be able to easily identify which exercises are practice quizzes vs. regular exercises

### ISSUE THREAD

**Issue by marcellamaki** [learningequality/kolibri#10440]
## Observed behavior
Noted by Francis in the Alpha-8 bug bash: 

"After signing is as a learner, many folders appeared and I couldn't tell which has the practice exercise. Is is possible to have like a thumbnail that would show that this is the latest practice quiz?"

## Context
Kolibri 0.16.0-alpha8

**Comment by rtibbles** [comment]
This was unimplemented from the original spec:

![image](https://user-images.githubusercontent.com/1680573/231226102-3bba8335-0388-46f8-bfe1-21a30e74986e.png)


Note that following conversation with @jtamiace, we decided to simplify the display for exercises to `Get X correct` in parallel to how it is displayed in the mastery bar.

So practice quizzes should be displayed with Practice quiz, and exercises should be displayed as `Get X correct`.

The two strings for this are found in commonCoreStrings: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/mixins/commonCoreStrings.js#L997

https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/mixins/commonCoreStrings.js#L1002

**Comment by Wck-iipi** [comment]
@marcellamaki @rtibbles Can I work on this issue?

**Comment by rtibbles** [comment]
Hi @Wck-iipi this might involve a bit of complexity, and the spec is a little implicit in terms of what the behaviour should be for different card types, so this may take some iteration to complete, so do be aware of that!

The main thing to be aware of is the different resource cards that are used in learn, so best to test fixes for this at a variety of screen breakpoints (using the browser dev tools to test on a mobile view, and larger screens), and on different pages of Learn.

For the HybridLearningContentCard, this kind of information is displayed in this component: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/CardThumbnail.vue

For the ResourceCard, it would be added in here: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/views/cards/ResourceCard.vue

There are some other cards used as well.

If you would like to take this on, definitely worth doing an exploration of which cards are in use, and then following up on this issue with ideas about your approaches.

**Comment by Wck-iipi** [comment]
@rtibbles I was thinking of three various solutions for this:
1. Instead of creating another card in thumbnail why don't we just change heading(LearningActivityLabel) Practice to Practice(Get x correct) or Practice(Practice Quiz) which will make it easier to distinguish between the different quizzes. This can be achieved by causing required changes in https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/composables/useLearningActivities.js. This can be done by appending required corestring to LearningActivites.PRACTICE and by comparing value of `contentNode.options.completion_criteria.threshold.mastery_model`. Interestingly enough, ResourceCard doesn't have options and gives null which might be a problem in implementing this.(LibraryAndChannelBrowserMainContent props doesn't contains option but value is null.)
2. I can add another card in HybridLearningContentCard by adding a CardLink inside the files. (Like you did in photos)
3. I can create a custom component that will just contain the text and put it in the thumbnail. (Like you did in photos)


I tried finding all the files that would require this and found CardList in bookmarks section only. Rest all of the cards that reference the exercises are HybridLearningContentCard only.
Please correct me if I have missed something or am wrong in something, thanks.

**Comment by marcellamaki** [comment]
@Wck-iipi - 

Thank your for your thoughts on this and the work you've done so far. I think for an initial solution, let's keep things as simple as possible. Currently, there is an existing component which exists in the card thumbnail, used in `HybridLearningContentCard` which manages the display of text called `LearningActivityDuration`. The most straightforward solution would be to just update that, for now. As for the text, I think it would be clearest to say only "Practice quiz" on the label for any activity that is a practice quiz, using the logic you describe. We have this in `commonCoreStrings`. It may require a bit of refactoring of some props to ensure that the data is available to do the content type check that you need.

Updating this component will reflect the changes on both large and small screens:

<img width="415" alt="Screenshot 2023-04-17 at 4 26 09 PM" src="https://user-images.githubusercontent.com/17235236/232602707-ecc727b9-ee91-4f9f-beb3-60fc868cc974.png">

 <img width="1413" alt="Screenshot 2023-04-17 at 4 26 33 PM" src="https://user-images.githubusercontent.com/17235236/232602723-9e48e60b-ab63-4fec-bbd7-fc5021fab6b6.png">




Please let me know if you have further questions, and thanks again for your interest and efforts here.

**Comment by Wck-iipi** [comment]
@marcellamaki Thank you for guiding me. I have added pull request related to this issue and I want to ask you if this is in the right direction. Instead of `options`, now I have used `assessmentmetadata`(as all cards contain this value as opposed to options). In this pull request, only practice quiz gets Practice Quiz label. Rest all of the others (m_of_n, get 100% correct, Get 2/3/5/10 correct) are of form 'Get x correct'. Is this ok(even for get 100% correct, which shows Get totalNumberOfQuestions correct)?
![image](https://user-images.githubusercontent.com/110763795/233449996-910af165-ed0f-41b8-902b-fb9776e45cda.png)

**Comment by rtibbles** [comment]
Yes, using the `Get x correct` form is fine - we decided that was simpler than giving a more complex explanation in such a condensed format.

For practice quizzes, it will still be necessary to do the check of the `options` property, because there are some practice quizzes (due some historic resource data inconsistencies) that do not have the mastery model set to 'quiz'.
